src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/AggregationManager.cs

```diff
@@ -148,14 +160,26 @@ public void Start()
             _initialInstrumentEnumerationComplete();
         }
 
+        public void Update()
+        {
+            using (MeterListener tempListener = new MeterListener())
```

Comment:
Can you add some comments for this? It's not immediately clear to me why we create a listener, Start, and then Dispose it.

Reply:
https://github.com/dotnet/runtime/pull/86504#discussion_r1207448693 - I can add comments though